src/main/java/org/gbif/ipt/action/manage/AutoPublishAction.java

```diff
@@ -176,4 +180,16 @@ private void populateDaysOfWeek() {
       daysOfWeek.put(dayOfWeek.getIdentifier(), getText("manage.autopublish." + dayOfWeek.getIdentifier()));
     }
   }
+
+  private void setServerTimeZone() {
+    req.setAttribute("serverTimeZone", TimeZone.getDefault().getDisplayName(false ,0));
```

Comment:
This looks misformated. Also, instead of `0` I would suggest using `TimeZone.SHORT` - it makes it clearer